docs/data-guidelines.md

```diff
@@ -12,6 +12,16 @@ If it's helpful to understanding the rule, summarize the rationale. Definitely c
 
 -- END TEMPLATE -->
 
+
+## Adding subfeatures
+
+Subfeatures should only be added to indicate a  _difference_ in browser compatibility versions from the parent feature; they should not be added just to indicate that a particular feature exists.
+
+For example, the `ImageData` interface was initially defined with `data`, `height` and `width` properties.
+As these properties were defined in the same versions as the parent feature [`ImageData`](https://github.com/mdn/browser-compat-data/blob/main/api/ImageData.json), they do not require a subfeature entry.
+However the `colorSpace` property, the constructor, and support for using the API in workers were added in later versions.
+This indicates a compatibility break, which is why a subfeature entry is required.
```

Comment:
```suggestion
For example, the `ImageData` interface was initially defined with `data`, `height` and `width` properties.
These do not require a subfeature entry because they were defined in the same versions as the parent feature [`ImageData`](https://github.com/mdn/browser-compat-data/blob/main/api/ImageData.json).

The subfeatures for the `colorSpace` property, the constructor, and support for using the API in workers, are required, because they appear in different versions than the original `ImageData` feature.
```